pkgs/by-name/le/lesspass-client/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lesspass-client";
+  version = "0.9.0";
+
+  src = fetchFromGitLab {
+    owner = "ogarcia";
+    repo = pname;
```

```suggestion
    repo = "lesspass-client";
```
use of pname like this is an antipattern